src/apps/template/appconstants.h

```diff
@@ -22,6 +22,33 @@ constexpr const char* MACOS_FALLBACK_APP_ID = "org.mozilla.macos.AppFooBar";
 constexpr const char* IOS_FALLBACK_APP_ID = "org.mozilla.ios.AppFooBar";
 #endif
 
+#if defined(MZ_WINDOWS)
```

Comment:
I feel like these should not be in the template unless they are things we expect *all* applications to provide, and I don't think it's appropriate to assume that all applications are going to support addons and cryptosettings.

Reply:
Why not? Why we should have not encrypted settings? Plus add-ons are a key component for content delivering. The suite app needs both components.

Reply:
I'm just thinking ahead for stuff like unit tests where we may not want to introduce platform-specific variability in the test runs.

Reply:
Why not? We run unit tests on all the platforms. In this way we increase the test coverage.